### ISSUE TITLE
Incorrect links in the Table of Contents sidebar

### ISSUE THREAD

**Issue by wjhopper** [moderndive/ModernDive_book#391]
There are a few incorrect links in the Table of Contents sidebar. For example, both the "10.5 Conclusion" link and the "9.6 Conclusion" links in the side bar take you to the Conclusion section of Chapter 1. 

I think this is because those section headings are ambiguous - there are multiple "Conclusion" sections in the book, one per chapter in fact. So, the link for the "10.5 Conclusion" section just takes you to the first section in the book titled "Conclusion" it can find, which happens to be in Chapter 1. 

I think the fix is to give each heading a disambiguating anchor. For example, the markdown for the Chapter 10 conclusion is currently
```
## Conclusion
```
and it should be something like
```
## Conclusion {#inference-conclusion}
```
for TOC linking to work correctly. 

I can make PR fixing these, but I know there is a bit of flux in the repo right now. Should I create the pull request against master, or refactor2, or something else?

**Comment by rudeboybert** [comment]
Hey @wjhopper, thanks for pointing this out. The content  on `master` is pretty much stable at this point (only a few cosmetic changes left), so we'd very much appreciate a PR there!

**Comment by ismayc** [comment]
moderndive.netlify.app and moderndive.com are now updated with current TOC sidebar links. (Via #395 and #393)

@rudeboybert Note that I had to do a manual travis Trigger build on `master` to get travis to build the book into `gh-pages`. I also had to do a manual deploy on Netlify to get moderndive.com to read from the updated `gh-pages`. It looks like GitHub is having problems syncing currently. 

Thanks again, @wjhopper!
